openraft/tests/fixtures/mod.rs

```diff
@@ -384,6 +395,12 @@ where
         self.isolated_nodes.lock().unwrap().insert(id);
     }
 
+    /// Make the network of the specified node unreachable.
+    #[tracing::instrument(level = "debug", skip(self))]
+    pub fn block_node(&self, id: C::NodeId) {
+        self.unreachable_nodes.lock().unwrap().insert(id);
```

Comment:
Inconsistent naming: rename `unreachable_nodes` to `unconnectable`, to describe exactly what this field does.

rename `block_node()` to something like `block_connect()`.